components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/Constants.java

```diff
@@ -120,4 +120,19 @@ private UserRegistrationConstants() {
 
         }
     }
+
+    public static class ErrorToi18nMappingConstants {
+
+        public static final String INVALID_CALLBACK_CALLBACK_NOT_MATCH = "invalid_callback_callback.not.match";
```

Comment:
Any reason to use `callback` twice? Can't we use `invalid_callback.not.match`.

Reply:
This is a combination of two variables. So we can't remove the repetition here.